04_numeronym/amitkrishna/numeronym_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var numeronymInputData = []struct {
+	n        []string // input
+	expected []string // expected result
+}{
+	{[]string{"accessibility", "Kubernetes", "abc"}, []string{"a11y", "K8s", "abc"}},
+	{[]string{"a", "b", "abc", "abcd"}, []string{"a", "b", "abc", "a2d"}},
```

Comment:
Add unicode tests, test empty string, spaces.

Reply:
Added the test